buildozer/targets/android.py

```diff
@@ -955,6 +955,12 @@ def execute_build_package(self, build_cmd):
         if extra_manifest_xml:
             cmd.append('--extra-manifest-xml="{}"'.format(open(extra_manifest_xml, 'rt').read()))
 
+        # support for extra-manifest-application
+        extra_manifest_application_entry = self.buildozer.config.getdefault(
+            'app', 'android.extra_manifest_application_entry', '')
+        if extra_manifest_application:
```

Comment:
incorrect variable name here... should be "extra_manifest_application_entry" and not "if extra_manifest_application"

Reply:
Yes, I see. Thx.
Updated.
![image](https://user-images.githubusercontent.com/1677773/218304447-57c248df-2066-411b-8b45-6c992ab5e98f.png)